doc/developer-guides/sw_design_guidelines.rst

```diff
@@ -64,7 +64,9 @@ below:
   All application constraints shall be guaranteed by external safety
   applications, such as Board Support Package, firmware, safety VM, or Hardware.
   The verification of application integration shall check whether the safety
-  application meets all application constraints.
+  application meets all application constraints, hyppervisor tester should verify
+  it during validation test. It is optional to do error checking for application
```

Comment:
```suggestion
  during validation test. It is optional to do error checking for application
```

Reply:
Thanks for your comment and review, will update it.